syntax/parse_test.go

```diff
@@ -257,6 +258,7 @@ func stripPos(err error) string {
 func treeString(n syntax.Node) string {
 	var buf bytes.Buffer
 	writeTree(&buf, reflect.ValueOf(n))
+	// Skip comments fields
```

Comment:
This comment doesn't belong here.  Delete it?

Reply:
Done